github3/pulls.py

```diff
@@ -178,10 +177,8 @@ def _update_attributes(self, pull):
         #: GitHub.com url for review comments (not a template)
         self.review_comments_url = pull.get('review_comments_url')
 
-        m = match('https?://[\w\d\-\.\:]+/(\S+)/(\S+)/(?:issues|pull)?/\d+',
-                  self.issue_url)
         #: Returns ('owner', 'repository') this issue was filed on.
-        self.repository = m.groups()
+        self.repository = self.base.repo
```

Comment:
Is `self.base.repo` a tuple or not? I'm guessing not, so you're breaking the API and not updating the documentation for the attribute.